src/entity/user/UserStreakAction.ts

```diff
@@ -16,7 +21,7 @@ export class UserStreakAction {
   @PrimaryColumn({ type: 'timestamptz', default: () => 'now()' })
   createdAt: Date;
 
-  @OneToOne(() => User, (user) => user.streak, {
+  @ManyToOne(() => User, (user) => user.streak, {
```

Comment:
This should also be `streakActions`?

Reply:
You might have to run migrations too for this one. Let me know if you need any help on that end.